django.utils.autoreload INFO 2023-08-17 13:01:22,114 autoreload 31232 7696 Watching for file changes with StatReloader
django.utils.autoreload INFO 2023-08-17 13:03:08,850 autoreload 1180 10424 Watching for file changes with StatReloader
django.server INFO 2023-08-17 13:03:38,630 basehttp 1180 14968 "GET / HTTP/1.1" 200 12583
django.server INFO 2023-08-17 13:03:39,189 basehttp 1180 14968 "GET /static/css/overideboot_size.css HTTP/1.1" 200 1693
django.server INFO 2023-08-17 13:03:39,191 basehttp 1180 14968 "GET /static/js/js_utils/getCookie.js HTTP/1.1" 200 515
django.server INFO 2023-08-17 13:03:39,195 basehttp 1180 14968 "GET /static/js/editableTable.js HTTP/1.1" 200 659
django.server INFO 2023-08-17 13:03:39,201 basehttp 1180 26016 "GET /static/css/table_wrap.css HTTP/1.1" 200 4733
django.server INFO 2023-08-17 13:03:39,204 basehttp 1180 5648 "GET /static/js/jquery-3.6.1.min.js HTTP/1.1" 200 89666
django.server INFO 2023-08-17 13:03:39,211 basehttp 1180 5648 "GET /static/js/loadModal.js HTTP/1.1" 200 348
django.server INFO 2023-08-17 13:03:39,212 basehttp 1180 30504 "GET /static/js/sorting_table.js HTTP/1.1" 200 890
django.server INFO 2023-08-17 13:03:39,212 basehttp 1180 14968 "GET /static/js/js_utils/dataTables.min.js HTTP/1.1" 200 102515
django.server INFO 2023-08-17 13:03:39,214 basehttp 1180 26016 "GET /static/js/js_utils/custom_option.js HTTP/1.1" 200 260
django.server INFO 2023-08-17 13:03:39,221 basehttp 1180 5648 "GET /static/js/resizable.js HTTP/1.1" 200 1691
django.server INFO 2023-08-17 13:03:39,223 basehttp 1180 26112 "GET /static/css/custom.css HTTP/1.1" 200 18869
django.server INFO 2023-08-17 13:03:39,225 basehttp 1180 29884 "GET /static/css/main.6a646761.css HTTP/1.1" 200 169393
django.server INFO 2023-08-17 13:03:39,724 basehttp 1180 29884 "GET /static/images/brand/CO-ADD_Logo_White_NoBackground_CMYK.png HTTP/1.1" 200 2856
django.server INFO 2023-08-17 13:03:39,724 basehttp 1180 5648 "GET /static/images/app/avatar.png HTTP/1.1" 200 40220
django.server INFO 2023-08-17 13:03:39,938 basehttp 1180 29884 "GET /static/images/app/CO-ADD_favicon.png HTTP/1.1" 200 3544
django.server INFO 2023-08-17 13:03:47,135 basehttp 1180 29884 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2023-08-17 13:03:47,557 basehttp 1180 29884 "GET /index/ HTTP/1.1" 200 19899
django.server INFO 2023-08-17 13:03:57,893 basehttp 1180 29884 "GET /user_profile/J.Zuegg HTTP/1.1" 200 20540
django.server INFO 2023-08-17 13:03:57,945 basehttp 1180 29884 "GET /static/js/filter_toggle.js HTTP/1.1" 200 172
django.server INFO 2023-08-17 13:04:09,739 basehttp 1180 29884 "GET /log_list/ HTTP/1.1" 200 108982
django.server INFO 2023-08-17 13:06:13,449 basehttp 1180 6996 "GET /log_list/?log_code=&log_proc=&log_type=&log_time=%3E2023&log_status=&log_user=&log_object=&log_desc= HTTP/1.1" 200 109760
django.server INFO 2023-08-17 13:06:13,507 basehttp 1180 6996 "GET /static/css/table_wrap.css HTTP/1.1" 304 0
django.server INFO 2023-08-17 13:06:16,818 basehttp 1180 6996 "GET /log_list/?csrfmiddlewaretoken=&Search_all_fields=&reset=True&pivot=False&paginate_by=50&order_by=&substructure=&log_code=&log_proc=&log_type=&log_time=&log_status=&log_user=&log_object=&log_desc= HTTP/1.1" 200 110230
django.server INFO 2023-08-17 13:06:20,619 basehttp 1180 6996 "GET /log_list/?log_code=&log_proc=&log_type=&log_time=%3C2023&log_status=&log_user=&log_object=&log_desc= HTTP/1.1" 200 109760
django.server INFO 2023-08-17 13:06:27,457 basehttp 1180 6996 "GET /log_list/?log_code=&log_proc=&log_type=&log_time=March&log_status=&log_user=&log_object=&log_desc= HTTP/1.1" 200 109741
django.server INFO 2023-08-17 13:06:33,564 basehttp 1180 6996 "GET /log_list/?log_code=&log_proc=&log_type=&log_time=May&log_status=&log_user=&log_object=&log_desc= HTTP/1.1" 200 109725
django.server INFO 2023-08-17 13:06:44,115 basehttp 1180 6996 "GET /log_list/?csrfmiddlewaretoken=&Search_all_fields=&reset=True&pivot=False&paginate_by=50&order_by=&substructure=&log_code=&log_proc=&log_type=&log_time=&log_status=&log_user=&log_object=&log_desc= HTTP/1.1" 200 110230
django.server INFO 2023-08-17 13:06:44,291 basehttp 1180 6996 - Broken pipe from ('127.0.0.1', 4193)
django.server INFO 2023-08-17 13:06:48,828 basehttp 1180 16788 "GET /log_list/?log_code=&log_proc=&log_type=&log_time=&log_status=Completed&log_user=&log_object=&log_desc= HTTP/1.1" 200 109793
django.server INFO 2023-08-17 13:06:53,201 basehttp 1180 16788 "GET /log_list/?csrfmiddlewaretoken=&Search_all_fields=&reset=True&pivot=False&paginate_by=50&order_by=&substructure=&log_code=&log_proc=&log_type=&log_time=&log_status=&log_user=&log_object=&log_desc= HTTP/1.1" 200 110230
django.server INFO 2023-08-17 13:07:17,168 basehttp 1180 16788 "GET /log_list/?csrfmiddlewaretoken=ZxiwkXT8Bem83eJ7YEqGclBb5ZrV1fMVD5PeAztKvHW8q8yPC4mCMqoY7R84UEzD&Search_all_fields=&reset=False&pivot=False&paginate_by=50&order_by=-Log+Time&substructure=&log_code=&log_proc=&log_type=&log_time=&log_status=&log_user=&log_object=&log_desc= HTTP/1.1" 200 106079
django.server INFO 2023-08-17 13:08:00,407 basehttp 1180 16788 "GET /log_list/?log_code=&log_proc=FG_0004&log_type=&log_time=&log_status=&log_user=&log_object=&log_desc= HTTP/1.1" 200 48269
django.server INFO 2023-08-17 13:08:04,209 basehttp 1180 16788 "GET /log_list/?csrfmiddlewaretoken=&Search_all_fields=&reset=True&pivot=False&paginate_by=50&order_by=&substructure=&log_code=&log_proc=&log_type=&log_time=&log_status=&log_user=&log_object=&log_desc= HTTP/1.1" 200 110230
django.server INFO 2023-08-17 13:08:10,155 basehttp 1180 16788 "GET /log_list/?log_code=&log_proc=Vitek&log_type=&log_time=&log_status=&log_user=&log_object=&log_desc= HTTP/1.1" 200 109693
django.server INFO 2023-08-17 13:08:17,333 basehttp 1180 16788 "GET /log_list/?csrfmiddlewaretoken=&Search_all_fields=&reset=True&pivot=False&paginate_by=50&order_by=&substructure=&log_code=&log_proc=&log_type=&log_time=&log_status=&log_user=&log_object=&log_desc= HTTP/1.1" 200 110230
django.utils.autoreload INFO 2023-08-17 13:14:15,249 autoreload 28164 2608 Watching for file changes with StatReloader
django.utils.autoreload INFO 2023-08-17 13:23:43,369 autoreload 24580 22212 Watching for file changes with StatReloader
django.server INFO 2023-08-17 13:23:49,413 basehttp 24580 14112 "GET / HTTP/1.1" 200 12584
django.server INFO 2023-08-17 13:23:49,729 basehttp 24580 27652 "GET /static/css/custom.css HTTP/1.1" 200 18869
django.server INFO 2023-08-17 13:23:49,732 basehttp 24580 14112 "GET /static/css/overideboot_size.css HTTP/1.1" 200 1693
django.server INFO 2023-08-17 13:23:49,736 basehttp 24580 27652 "GET /static/css/table_wrap.css HTTP/1.1" 200 4733
django.server INFO 2023-08-17 13:23:49,740 basehttp 24580 14112 "GET /static/js/jquery-3.6.1.min.js HTTP/1.1" 200 89666
django.server INFO 2023-08-17 13:23:49,741 basehttp 24580 27652 "GET /static/js/sorting_table.js HTTP/1.1" 200 890
django.server INFO 2023-08-17 13:23:49,743 basehttp 24580 14112 "GET /static/js/js_utils/getCookie.js HTTP/1.1" 200 515
django.server INFO 2023-08-17 13:23:49,749 basehttp 24580 27652 "GET /static/js/editableTable.js HTTP/1.1" 200 659
django.server INFO 2023-08-17 13:23:49,750 basehttp 24580 14112 "GET /static/js/js_utils/custom_option.js HTTP/1.1" 200 260
django.server INFO 2023-08-17 13:23:49,750 basehttp 24580 23348 "GET /static/js/loadModal.js HTTP/1.1" 200 348
django.server INFO 2023-08-17 13:23:49,753 basehttp 24580 4856 "GET /static/js/js_utils/dataTables.min.js HTTP/1.1" 200 102515
django.server INFO 2023-08-17 13:23:49,756 basehttp 24580 27652 "GET /static/js/resizable.js HTTP/1.1" 200 1691
django.server INFO 2023-08-17 13:23:49,757 basehttp 24580 23348 "GET /static/images/brand/CO-ADD_Logo_White_NoBackground_CMYK.png HTTP/1.1" 200 2856
django.server INFO 2023-08-17 13:23:49,758 basehttp 24580 14112 "GET /static/images/app/avatar.png HTTP/1.1" 200 40220
django.server INFO 2023-08-17 13:23:49,769 basehttp 24580 5028 "GET /static/css/main.6a646761.css HTTP/1.1" 200 169393
django.server INFO 2023-08-17 13:23:50,493 basehttp 24580 5028 "GET /static/images/app/CO-ADD_favicon.png HTTP/1.1" 200 3544
django.server INFO 2023-08-17 13:23:57,366 basehttp 24580 5028 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2023-08-17 13:23:57,821 basehttp 24580 5028 "GET /index/ HTTP/1.1" 200 19900
django.server INFO 2023-08-17 13:24:03,836 basehttp 24580 5028 "GET /dorganism/organism_card HTTP/1.1" 200 179919
django.server INFO 2023-08-17 13:24:03,859 basehttp 24580 5028 "GET /static/js/filter_toggle.js HTTP/1.1" 200 172
django.server INFO 2023-08-17 13:24:10,294 basehttp 24580 5028 "GET /dorganism/organism_card?csrfmiddlewaretoken=dWSMrxPt88ZDGCYdHXGF3AZgwX8QYM47vpJ250UdqXeDPtt39IGP9QZCisUtQHbk&applysingle=Apply&Search_all_fields=0517&reset=False&pivot=False&paginate_by=50&order_by=&substructure=&Class=&ID=&Name=&Strain=&Notes=&MTA= HTTP/1.1" 200 42340
django.server INFO 2023-08-17 13:24:12,076 basehttp 24580 5028 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 95549
django.server INFO 2023-08-17 13:24:12,107 basehttp 24580 5028 "GET /static/js/search_organism.js HTTP/1.1" 200 2506
django.server INFO 2023-08-17 13:24:12,108 basehttp 24580 14112 "GET /static/js/create_childtable.js HTTP/1.1" 200 6724
django.server INFO 2023-08-17 13:24:12,123 basehttp 24580 5028 "GET /static/js/destroy_childtable.js HTTP/1.1" 200 201
django.server INFO 2023-08-17 13:24:12,124 basehttp 24580 14112 "GET /static/js/childtable.js HTTP/1.1" 200 750
django.server INFO 2023-08-17 13:24:12,141 basehttp 24580 14112 "GET /static/js/ajax_simpleupload.js HTTP/1.1" 200 1244
django.server INFO 2023-08-17 13:24:41,702 basehttp 24580 14112 "POST /dorganism/createbatchimg/GN_0517 HTTP/1.1" 200 46
django.server INFO 2023-08-17 13:24:44,134 basehttp 24580 14112 "POST /dorganism/createbatchimg/GN_0517 HTTP/1.1" 302 0
django.server INFO 2023-08-17 13:24:44,552 basehttp 24580 14112 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 99349
django.server INFO 2023-08-17 13:24:44,651 basehttp 24580 14112 "GET /uploads/images/orgbatch/GN_0517.jpeg HTTP/1.1" 200 103757
django.server INFO 2023-08-17 13:28:21,253 basehttp 24580 14112 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 99352
django.server INFO 2023-08-17 13:28:21,275 basehttp 24580 14112 "GET /static/css/table_wrap.css HTTP/1.1" 304 0
django.server INFO 2023-08-17 13:28:21,353 basehttp 24580 5028 "GET /uploads/images/orgbatch/GN_0517.jpeg HTTP/1.1" 304 0
django.utils.autoreload INFO 2023-08-17 13:28:57,007 autoreload 31692 23600 Watching for file changes with StatReloader
django.server INFO 2023-08-17 13:29:02,548 basehttp 31692 14068 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 99352
django.server INFO 2023-08-17 13:29:02,651 basehttp 31692 14068 "GET /uploads/images/orgbatch/GN_0517.jpeg HTTP/1.1" 304 0
django.server INFO 2023-08-17 13:31:13,770 basehttp 31692 14068 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 99337
django.server INFO 2023-08-17 13:31:13,899 basehttp 31692 14068 "GET /uploads/images/orgbatch/GN_0517.jpeg HTTP/1.1" 304 0
django.server INFO 2023-08-17 13:32:03,586 basehttp 31692 14068 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 99355
django.server INFO 2023-08-17 13:32:03,775 basehttp 31692 14068 "GET /uploads/images/orgbatch/GN_0517.jpeg HTTP/1.1" 304 0
django.server INFO 2023-08-17 13:33:03,026 basehttp 31692 14068 "POST /dorganism/createbatchimg/GN_0517 HTTP/1.1" 200 46
django.server INFO 2023-08-17 13:33:28,538 basehttp 31692 14068 "POST /dorganism/createbatchimg/GN_0517 HTTP/1.1" 302 0
django.server INFO 2023-08-17 13:33:29,041 basehttp 31692 14068 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 103166
django.server INFO 2023-08-17 13:33:29,078 basehttp 31692 21988 "GET /static/css/table_wrap.css HTTP/1.1" 304 0
django.server INFO 2023-08-17 13:33:29,223 basehttp 31692 21988 "GET /uploads/images/orgbatch/GN_0517.jpeg HTTP/1.1" 304 0
django.server INFO 2023-08-17 13:33:29,648 basehttp 31692 14068 "GET /uploads/images/orgbatch/GN_0518.jpeg HTTP/1.1" 200 89022
django.server INFO 2023-08-17 13:34:38,256 basehttp 31692 14068 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 103080
django.server INFO 2023-08-17 13:34:38,342 basehttp 31692 21988 "GET /uploads/images/orgbatch/GN_0517.jpeg HTTP/1.1" 304 0
django.server INFO 2023-08-17 13:34:38,406 basehttp 31692 14068 "GET /uploads/images/orgbatch/GN_0518.jpeg HTTP/1.1" 304 0
django.server INFO 2023-08-17 13:36:08,410 basehttp 31692 14068 "POST /dorganism/createbatchimg/GN_0517 HTTP/1.1" 200 46
django.server INFO 2023-08-17 13:36:36,243 basehttp 31692 14068 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 103072
django.server INFO 2023-08-17 13:36:36,360 basehttp 31692 21988 "GET /uploads/images/orgbatch/GN_0517.jpeg HTTP/1.1" 304 0
django.request WARNING 2023-08-17 13:36:36,476 log 31692 14068 The request's session was deleted before the request completed. The user may have logged out in a concurrent request, for example.: /uploads/images/orgbatch/GN_0518.jpeg
Traceback (most recent call last):
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.SerializationFailure: could not serialize access due to concurrent update


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\contrib\sessions\backends\db.py", line 86, in save
    obj.save(
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\models\base.py", line 814, in save
    self.save_base(
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\models\base.py", line 877, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\models\base.py", line 990, in _save_table
    updated = self._do_update(
              ^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\models\base.py", line 1054, in _do_update
    return filtered._update(values) > 0
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\models\query.py", line 1231, in _update
    return query.get_compiler(self.db).execute_sql(CURSOR)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\models\sql\compiler.py", line 1984, in execute_sql
    cursor = super().execute_sql(result_type)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\models\sql\compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\backends\utils.py", line 102, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\backends\utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\backends\utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\backends\utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.OperationalError: could not serialize access due to concurrent update


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\contrib\sessions\middleware.py", line 59, in process_response
    request.session.save()
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\contrib\sessions\backends\db.py", line 95, in save
    raise UpdateError
django.contrib.sessions.backends.base.UpdateError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\utils\deprecation.py", line 136, in __call__
    response = self.process_response(request, response)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\contrib\sessions\middleware.py", line 61, in process_response
    raise SessionInterrupted(
django.contrib.sessions.exceptions.SessionInterrupted: The request's session was deleted before the request completed. The user may have logged out in a concurrent request, for example.
django.server WARNING 2023-08-17 13:36:36,486 basehttp 31692 14068 "GET /uploads/images/orgbatch/GN_0518.jpeg HTTP/1.1" 400 169131
django.server INFO 2023-08-17 13:41:37,264 basehttp 31692 8460 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 103072
django.server INFO 2023-08-17 13:41:37,295 basehttp 31692 8460 "GET /static/css/table_wrap.css HTTP/1.1" 304 0
django.server INFO 2023-08-17 13:41:37,378 basehttp 31692 14940 "GET /uploads/images/orgbatch/GN_0517.jpeg HTTP/1.1" 304 0
django.server INFO 2023-08-17 13:41:37,380 basehttp 31692 11072 "GET /uploads/images/orgbatch/GN_0518.jpeg HTTP/1.1" 200 89022
django.server INFO 2023-08-17 13:46:38,167 basehttp 31692 15968 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 103072
django.server INFO 2023-08-17 13:46:38,260 basehttp 31692 23420 "GET /uploads/images/orgbatch/GN_0517.jpeg HTTP/1.1" 304 0
django.server INFO 2023-08-17 13:46:38,273 basehttp 31692 15968 "GET /uploads/images/orgbatch/GN_0518.jpeg HTTP/1.1" 304 0
django.server INFO 2023-08-17 13:47:51,179 basehttp 31692 15968 "POST /dorganism/deleteBatimg/22 HTTP/1.1" 302 0
django.server INFO 2023-08-17 13:47:51,607 basehttp 31692 15968 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 99308
django.server INFO 2023-08-17 13:49:25,048 basehttp 31692 15968 "POST /dorganism/createbatchimg/GN_0517 HTTP/1.1" 200 46
django.utils.autoreload INFO 2023-08-17 13:50:32,534 autoreload 31692 23600 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 13:50:37,082 autoreload 26240 17068 Watching for file changes with StatReloader
django.utils.autoreload INFO 2023-08-17 13:51:16,571 autoreload 26240 17068 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 13:51:21,273 autoreload 13496 24624 Watching for file changes with StatReloader
django.utils.autoreload INFO 2023-08-17 13:51:26,358 autoreload 13496 24624 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 13:51:30,409 autoreload 16568 27328 Watching for file changes with StatReloader
django.utils.autoreload INFO 2023-08-17 13:51:37,149 autoreload 16568 27328 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 13:51:40,711 autoreload 22852 22896 Watching for file changes with StatReloader
django.utils.autoreload INFO 2023-08-17 13:51:52,518 autoreload 22852 22896 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 13:51:56,126 autoreload 4528 24328 Watching for file changes with StatReloader
django.utils.autoreload INFO 2023-08-17 13:52:03,682 autoreload 4528 24328 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 13:52:08,930 autoreload 18460 10280 Watching for file changes with StatReloader
django.utils.autoreload INFO 2023-08-17 13:52:15,489 autoreload 18460 10280 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 13:52:19,854 autoreload 26292 9868 Watching for file changes with StatReloader
django.utils.autoreload INFO 2023-08-17 13:52:26,871 autoreload 26292 9868 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 13:52:30,986 autoreload 30128 18864 Watching for file changes with StatReloader
django.utils.autoreload INFO 2023-08-17 13:52:42,784 autoreload 30128 18864 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 13:52:47,034 autoreload 27824 25064 Watching for file changes with StatReloader
django.utils.autoreload INFO 2023-08-17 13:52:56,101 autoreload 27824 25064 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 13:53:00,671 autoreload 28796 11472 Watching for file changes with StatReloader
django.utils.autoreload INFO 2023-08-17 13:53:11,775 autoreload 28796 11472 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 13:54:38,814 autoreload 2464 6776 Watching for file changes with StatReloader
django.server INFO 2023-08-17 13:54:44,910 basehttp 2464 16468 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 99313
django.server INFO 2023-08-17 13:54:44,981 basehttp 2464 16468 "GET /static/css/table_wrap.css HTTP/1.1" 304 0
django.server INFO 2023-08-17 13:54:45,301 basehttp 2464 16468 "GET /uploads/images/orgbatch/GN_0518.jpeg HTTP/1.1" 304 0
django.utils.autoreload INFO 2023-08-17 13:55:19,964 autoreload 2464 6776 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 13:55:24,496 autoreload 10588 4256 Watching for file changes with StatReloader
django.utils.autoreload INFO 2023-08-17 13:55:29,476 autoreload 10588 4256 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 13:55:32,003 autoreload 13212 18984 Watching for file changes with StatReloader
django.utils.autoreload INFO 2023-08-17 13:56:18,192 autoreload 13212 18984 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 13:56:22,579 autoreload 29796 24736 Watching for file changes with StatReloader
django.utils.autoreload INFO 2023-08-17 13:56:28,406 autoreload 29796 24736 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 13:56:32,378 autoreload 5376 23316 Watching for file changes with StatReloader
django.utils.autoreload INFO 2023-08-17 13:56:40,025 autoreload 5376 23316 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 13:56:43,206 autoreload 7132 4632 Watching for file changes with StatReloader
django.utils.autoreload INFO 2023-08-17 13:56:51,458 autoreload 7132 4632 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 13:56:54,960 autoreload 25736 10124 Watching for file changes with StatReloader
django.utils.autoreload INFO 2023-08-17 13:56:59,806 autoreload 25736 10124 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 13:57:03,282 autoreload 22304 12540 Watching for file changes with StatReloader
django.utils.autoreload INFO 2023-08-17 13:57:12,926 autoreload 22304 12540 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 13:57:16,019 autoreload 8172 27232 Watching for file changes with StatReloader
django.utils.autoreload INFO 2023-08-17 13:57:21,885 autoreload 8172 27232 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 13:57:24,658 autoreload 26972 22212 Watching for file changes with StatReloader
django.utils.autoreload INFO 2023-08-17 13:57:29,379 autoreload 26972 22212 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 13:57:32,411 autoreload 20488 13160 Watching for file changes with StatReloader
django.utils.autoreload INFO 2023-08-17 13:57:36,792 autoreload 20488 13160 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 13:57:39,552 autoreload 30660 21484 Watching for file changes with StatReloader
django.utils.autoreload INFO 2023-08-17 13:58:33,041 autoreload 21508 30100 Watching for file changes with StatReloader
django.server INFO 2023-08-17 13:58:45,028 basehttp 21508 17488 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 99318
django.server INFO 2023-08-17 13:58:45,230 basehttp 21508 17488 "GET /uploads/images/orgbatch/GN_0518.jpeg HTTP/1.1" 304 0
django.server INFO 2023-08-17 13:58:46,973 basehttp 21508 17488 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 99318
django.server INFO 2023-08-17 13:59:01,936 basehttp 21508 17488 "POST /dorganism/createbatchimg/GN_0517 HTTP/1.1" 200 46
django.server INFO 2023-08-17 13:59:08,955 basehttp 21508 17488 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 99318
django.utils.autoreload INFO 2023-08-17 14:33:23,424 autoreload 26728 29860 Watching for file changes with StatReloader
django.server INFO 2023-08-17 14:33:29,439 basehttp 26728 10280 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 103102
django.server INFO 2023-08-17 14:33:29,482 basehttp 26728 10280 "GET /static/css/table_wrap.css HTTP/1.1" 304 0
django.server INFO 2023-08-17 14:33:29,721 basehttp 26728 12644 "GET /uploads/images/orgbatch/GN_0518.jpeg HTTP/1.1" 304 0
django.request WARNING 2023-08-17 14:33:29,749 log 26728 10280 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 14:33:29,750 basehttp 26728 10280 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.utils.autoreload INFO 2023-08-17 14:35:51,115 autoreload 26728 29860 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 14:35:53,452 autoreload 21120 22008 Watching for file changes with StatReloader
django.server INFO 2023-08-17 14:36:23,047 basehttp 21120 18460 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 103112
django.request WARNING 2023-08-17 14:36:23,228 log 21120 18460 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 14:36:23,229 basehttp 21120 18460 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.server INFO 2023-08-17 14:36:24,333 basehttp 21120 18460 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 103112
django.request WARNING 2023-08-17 14:36:24,474 log 21120 18460 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 14:36:24,475 basehttp 21120 18460 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.server INFO 2023-08-17 14:36:38,331 basehttp 21120 18460 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 103112
django.request WARNING 2023-08-17 14:36:38,468 log 21120 18460 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 14:36:38,470 basehttp 21120 18460 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.utils.autoreload INFO 2023-08-17 14:36:53,061 autoreload 21120 22008 C:\Data\A02_WorkDB\03_Django\adjCOADD\dorganism\forms.py changed, reloading.
django.utils.autoreload INFO 2023-08-17 14:36:55,685 autoreload 22052 8568 Watching for file changes with StatReloader
django.server INFO 2023-08-17 14:36:57,986 basehttp 22052 28732 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 103112
django.request WARNING 2023-08-17 14:36:58,123 log 22052 28732 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 14:36:58,123 basehttp 22052 28732 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.server INFO 2023-08-17 14:37:10,260 basehttp 22052 28732 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 103112
django.request WARNING 2023-08-17 14:37:10,379 log 22052 28732 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 14:37:10,380 basehttp 22052 28732 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.server INFO 2023-08-17 14:43:34,036 basehttp 22052 3364 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 103112
django.server INFO 2023-08-17 14:43:34,163 basehttp 22052 3364 "GET /uploads/images/orgbatch/GN_0518.jpeg HTTP/1.1" 304 0
django.request WARNING 2023-08-17 14:43:34,170 log 22052 19248 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 14:43:34,170 basehttp 22052 19248 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.request WARNING 2023-08-17 14:43:34,359 log 22052 19248 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 14:43:34,359 basehttp 22052 19248 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.server INFO 2023-08-17 14:48:38,153 basehttp 22052 20752 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 103112
django.server INFO 2023-08-17 14:48:39,789 basehttp 22052 20752 "GET /static/css/table_wrap.css HTTP/1.1" 304 0
django.request WARNING 2023-08-17 14:48:39,937 log 22052 19708 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 14:48:39,948 basehttp 22052 19708 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.request WARNING 2023-08-17 14:48:41,996 log 22052 19708 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 14:48:41,996 basehttp 22052 19708 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.server INFO 2023-08-17 14:53:43,037 basehttp 22052 22664 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 103112
django.server INFO 2023-08-17 14:53:43,160 basehttp 22052 22664 "GET /uploads/images/orgbatch/GN_0518.jpeg HTTP/1.1" 304 0
django.request WARNING 2023-08-17 14:53:43,183 log 22052 31488 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 14:53:43,183 basehttp 22052 31488 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.server INFO 2023-08-17 14:58:43,971 basehttp 22052 21204 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 103112
django.request WARNING 2023-08-17 14:58:44,076 log 22052 21204 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 14:58:44,076 basehttp 22052 21204 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.server INFO 2023-08-17 15:03:45,067 basehttp 22052 21624 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 103112
django.server INFO 2023-08-17 15:03:45,105 basehttp 22052 21624 "GET /static/css/table_wrap.css HTTP/1.1" 304 0
django.request WARNING 2023-08-17 15:03:45,177 log 22052 23364 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 15:03:45,178 basehttp 22052 23364 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.server INFO 2023-08-17 15:03:45,183 basehttp 22052 23996 "GET /uploads/images/orgbatch/GN_0518.jpeg HTTP/1.1" 304 0
django.server INFO 2023-08-17 15:08:45,972 basehttp 22052 31040 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 103112
django.request WARNING 2023-08-17 15:08:46,082 log 22052 31040 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 15:08:46,082 basehttp 22052 31040 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.server INFO 2023-08-17 15:13:46,996 basehttp 22052 26672 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 103112
django.server INFO 2023-08-17 15:13:47,129 basehttp 22052 26672 "GET /uploads/images/orgbatch/GN_0518.jpeg HTTP/1.1" 304 0
django.request WARNING 2023-08-17 15:13:47,133 log 22052 9660 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 15:13:47,134 basehttp 22052 9660 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.server INFO 2023-08-17 15:18:47,977 basehttp 22052 9308 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 103112
django.server INFO 2023-08-17 15:18:48,014 basehttp 22052 9308 "GET /static/css/table_wrap.css HTTP/1.1" 304 0
django.request WARNING 2023-08-17 15:18:48,083 log 22052 28784 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 15:18:48,083 basehttp 22052 28784 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.server INFO 2023-08-17 15:23:48,956 basehttp 22052 23316 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 103112
django.server INFO 2023-08-17 15:23:49,086 basehttp 22052 23316 "GET /uploads/images/orgbatch/GN_0518.jpeg HTTP/1.1" 304 0
django.request WARNING 2023-08-17 15:23:49,112 log 22052 8136 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 15:23:49,113 basehttp 22052 8136 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.server INFO 2023-08-17 15:28:50,025 basehttp 22052 26468 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 103112
django.request WARNING 2023-08-17 15:28:50,149 log 22052 26468 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 15:28:50,151 basehttp 22052 26468 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.server INFO 2023-08-17 15:30:08,031 basehttp 22052 26468 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 106920
django.request WARNING 2023-08-17 15:30:08,143 log 22052 26468 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 15:30:08,144 basehttp 22052 26468 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.request WARNING 2023-08-17 15:30:08,176 log 22052 30612 Not Found: /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg
django.server WARNING 2023-08-17 15:30:08,177 basehttp 22052 30612 "GET /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg HTTP/1.1" 404 6501
django.server INFO 2023-08-17 15:30:28,799 basehttp 22052 30612 "GET /dorganism/organism/GN_0518 HTTP/1.1" 200 94850
django.request WARNING 2023-08-17 15:30:28,905 log 22052 30612 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518_01.jpeg
django.server WARNING 2023-08-17 15:30:28,906 basehttp 22052 30612 "GET /uploads/images/orgbatch/GN_0500/GN_0518_01.jpeg HTTP/1.1" 404 6501
django.server INFO 2023-08-17 15:33:42,773 basehttp 22052 30612 "GET /dorganism/organism/GN_0518 HTTP/1.1" 200 94852
django.request WARNING 2023-08-17 15:33:42,900 log 22052 30612 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518_01.jpeg
django.server WARNING 2023-08-17 15:33:42,901 basehttp 22052 30612 "GET /uploads/images/orgbatch/GN_0500/GN_0518_01.jpeg HTTP/1.1" 404 6501
django.server INFO 2023-08-17 15:37:17,391 basehttp 22052 30612 "GET /static/css/table_wrap.css HTTP/1.1" 304 0
django.server INFO 2023-08-17 15:39:45,279 basehttp 22052 30612 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 106946
django.server INFO 2023-08-17 15:39:45,440 basehttp 22052 30612 "GET /uploads/images/orgbatch/GN_0518.jpeg HTTP/1.1" 304 0
django.request WARNING 2023-08-17 15:39:45,455 log 22052 17932 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 15:39:45,456 basehttp 22052 17932 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.request WARNING 2023-08-17 15:39:45,873 log 22052 17932 Not Found: /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg
django.server WARNING 2023-08-17 15:39:45,874 basehttp 22052 17932 "GET /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg HTTP/1.1" 404 6501
django.request WARNING 2023-08-17 15:39:46,167 log 22052 17932 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 15:39:46,169 basehttp 22052 17932 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.request WARNING 2023-08-17 15:39:46,170 log 22052 30612 Not Found: /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg
django.server WARNING 2023-08-17 15:39:46,171 basehttp 22052 30612 "GET /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg HTTP/1.1" 404 6501
django.server INFO 2023-08-17 15:41:49,424 basehttp 22052 30612 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 107040
django.request WARNING 2023-08-17 15:41:49,555 log 22052 30612 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 15:41:49,556 basehttp 22052 30612 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.request WARNING 2023-08-17 15:41:49,626 log 22052 17932 Not Found: /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg
django.server WARNING 2023-08-17 15:41:49,626 basehttp 22052 17932 "GET /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg HTTP/1.1" 404 6501
django.request WARNING 2023-08-17 15:41:50,049 log 22052 30612 Not Found: /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg
django.server WARNING 2023-08-17 15:41:50,050 basehttp 22052 30612 "GET /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg HTTP/1.1" 404 6501
django.request WARNING 2023-08-17 15:41:50,064 log 22052 17932 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 15:41:50,064 basehttp 22052 17932 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.request WARNING 2023-08-17 15:43:21,113 log 22052 17932 Not Found: /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg
django.server WARNING 2023-08-17 15:43:21,114 basehttp 22052 17932 "GET //uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg HTTP/1.1" 404 6501
django.request WARNING 2023-08-17 15:43:21,320 log 22052 17932 Not Found: /favicon.ico
django.server WARNING 2023-08-17 15:43:21,321 basehttp 22052 17932 "GET /favicon.ico HTTP/1.1" 404 6167
django.request WARNING 2023-08-17 15:43:39,800 log 22052 17932 Not Found: /uploads/images/orgbatch/
django.server WARNING 2023-08-17 15:43:39,802 basehttp 22052 17932 "GET //uploads/images/orgbatch/ HTTP/1.1" 404 6374
django.request WARNING 2023-08-17 15:45:35,258 log 22052 17932 Not Found: /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg
django.server WARNING 2023-08-17 15:45:35,258 basehttp 22052 17932 "GET /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg HTTP/1.1" 404 6501
django.request WARNING 2023-08-17 15:46:22,289 log 22052 17932 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518_01.jpeg
django.server WARNING 2023-08-17 15:46:22,290 basehttp 22052 17932 "GET /uploads/images/orgbatch/GN_0500/GN_0518_01.jpeg HTTP/1.1" 404 6501
django.request WARNING 2023-08-17 15:47:43,338 log 22052 17932 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518_01.jpeg
django.server WARNING 2023-08-17 15:47:43,339 basehttp 22052 17932 "GET //uploads/images/orgbatch/GN_0500/GN_0518_01.jpeg HTTP/1.1" 404 6501
django.request WARNING 2023-08-17 15:51:41,735 log 22052 17932 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518_01.jpeg
django.server WARNING 2023-08-17 15:51:41,736 basehttp 22052 17932 "GET //uploads/images/orgbatch/GN_0500/GN_0518_01.jpeg HTTP/1.1" 404 6501
django.server INFO 2023-08-17 15:51:45,695 basehttp 22052 17932 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 107112
django.request WARNING 2023-08-17 15:51:45,849 log 22052 17932 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 15:51:45,850 basehttp 22052 17932 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.request WARNING 2023-08-17 15:51:45,894 log 22052 12604 Not Found: /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg
django.server WARNING 2023-08-17 15:51:45,895 basehttp 22052 12604 "GET /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg HTTP/1.1" 404 6501
django.request WARNING 2023-08-17 15:51:46,311 log 22052 12604 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 15:51:46,311 basehttp 22052 12604 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.request WARNING 2023-08-17 15:51:46,312 log 22052 17932 Not Found: /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg
django.server WARNING 2023-08-17 15:51:46,313 basehttp 22052 17932 "GET /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg HTTP/1.1" 404 6501
django.server INFO 2023-08-17 15:51:47,834 basehttp 22052 17932 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 107112
django.request WARNING 2023-08-17 15:51:48,010 log 22052 17932 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 15:51:48,011 basehttp 22052 17932 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.request WARNING 2023-08-17 15:51:48,014 log 22052 12604 Not Found: /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg
django.server WARNING 2023-08-17 15:51:48,014 basehttp 22052 12604 "GET /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg HTTP/1.1" 404 6501
django.request WARNING 2023-08-17 15:51:48,400 log 22052 12604 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 15:51:48,401 basehttp 22052 12604 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.request WARNING 2023-08-17 15:51:48,495 log 22052 17932 The request's session was deleted before the request completed. The user may have logged out in a concurrent request, for example.: /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg
Traceback (most recent call last):
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.SerializationFailure: could not serialize access due to concurrent update


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\contrib\sessions\backends\db.py", line 86, in save
    obj.save(
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\models\base.py", line 814, in save
    self.save_base(
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\models\base.py", line 877, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\models\base.py", line 990, in _save_table
    updated = self._do_update(
              ^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\models\base.py", line 1054, in _do_update
    return filtered._update(values) > 0
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\models\query.py", line 1231, in _update
    return query.get_compiler(self.db).execute_sql(CURSOR)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\models\sql\compiler.py", line 1984, in execute_sql
    cursor = super().execute_sql(result_type)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\models\sql\compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\backends\utils.py", line 102, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\backends\utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\backends\utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\backends\utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.OperationalError: could not serialize access due to concurrent update


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\contrib\sessions\middleware.py", line 59, in process_response
    request.session.save()
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\contrib\sessions\backends\db.py", line 95, in save
    raise UpdateError
django.contrib.sessions.backends.base.UpdateError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\utils\deprecation.py", line 136, in __call__
    response = self.process_response(request, response)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\contrib\sessions\middleware.py", line 61, in process_response
    raise SessionInterrupted(
django.contrib.sessions.exceptions.SessionInterrupted: The request's session was deleted before the request completed. The user may have logged out in a concurrent request, for example.
django.server WARNING 2023-08-17 15:51:48,500 basehttp 22052 17932 "GET /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg HTTP/1.1" 400 169184
django.request WARNING 2023-08-17 15:52:14,613 log 22052 12604 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518_01.jpeg
django.server WARNING 2023-08-17 15:52:14,614 basehttp 22052 12604 "GET /uploads/images/orgbatch/GN_0500/GN_0518_01.jpeg HTTP/1.1" 404 6501
django.request WARNING 2023-08-17 15:54:29,125 log 22052 12604 Not Found: /uploads/orgbatch/GN_0500/GN_0518_01.jpeg
django.server WARNING 2023-08-17 15:54:29,126 basehttp 22052 12604 "GET /uploads/orgbatch/GN_0500/GN_0518_01.jpeg HTTP/1.1" 404 6473
django.request WARNING 2023-08-17 15:54:33,137 log 22052 12604 Not Found: /uploads/GN_0500/GN_0518_01.jpeg
django.server WARNING 2023-08-17 15:54:33,137 basehttp 22052 12604 "GET /uploads/GN_0500/GN_0518_01.jpeg HTTP/1.1" 404 6437
django.request WARNING 2023-08-17 15:54:45,560 log 22052 12604 Not Found: /uploads/images/orgbatch/GN_0518_01.jpeg
django.server WARNING 2023-08-17 15:54:45,561 basehttp 22052 12604 "GET /uploads/images/orgbatch/GN_0518_01.jpeg HTTP/1.1" 404 6469
django.server INFO 2023-08-17 15:55:05,921 basehttp 22052 12604 "GET /uploads/images/orgbatch/GN_0518.jpeg HTTP/1.1" 304 0
django.request WARNING 2023-08-17 15:55:38,852 log 22052 12604 Not Found: /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg
django.server WARNING 2023-08-17 15:55:38,853 basehttp 22052 12604 "GET /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg HTTP/1.1" 404 6501
django.request WARNING 2023-08-17 15:55:43,525 log 22052 12604 Not Found: /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg
django.server WARNING 2023-08-17 15:55:43,525 basehttp 22052 12604 "GET /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg HTTP/1.1" 404 6501
django.server INFO 2023-08-17 15:56:59,672 basehttp 22052 12604 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 107112
django.server INFO 2023-08-17 15:56:59,734 basehttp 22052 12604 "GET /static/css/table_wrap.css HTTP/1.1" 304 0
django.server INFO 2023-08-17 15:56:59,799 basehttp 22052 5576 "GET /uploads/images/orgbatch/GN_0518.jpeg HTTP/1.1" 304 0
django.request WARNING 2023-08-17 15:56:59,809 log 22052 12604 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 15:56:59,810 basehttp 22052 12604 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.request WARNING 2023-08-17 15:57:00,049 log 22052 12604 Not Found: /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg
django.server WARNING 2023-08-17 15:57:00,050 basehttp 22052 12604 "GET /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg HTTP/1.1" 404 6501
django.request WARNING 2023-08-17 15:57:00,090 log 22052 5576 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 15:57:00,090 basehttp 22052 5576 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.request WARNING 2023-08-17 15:58:12,115 log 22052 5576 Not Found: /uploads/images/orgbatch/GN_0517_01.jpeg
django.server WARNING 2023-08-17 15:58:12,116 basehttp 22052 5576 "GET /uploads/images/orgbatch/GN_0517_01.jpeg HTTP/1.1" 404 6469
django.server INFO 2023-08-17 16:02:00,710 basehttp 22052 5576 "GET /dorganism/organism/GN_0517 HTTP/1.1" 200 107112
django.request WARNING 2023-08-17 16:02:00,878 log 22052 5576 Not Found: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
django.server WARNING 2023-08-17 16:02:00,879 basehttp 22052 5576 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 404 6489
django.request WARNING 2023-08-17 16:02:00,885 log 22052 31484 Not Found: /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg
django.server WARNING 2023-08-17 16:02:00,886 basehttp 22052 31484 "GET /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg HTTP/1.1" 404 6501
django.request WARNING 2023-08-17 16:02:01,371 log 22052 5576 Not Found: /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg
django.server WARNING 2023-08-17 16:02:01,385 basehttp 22052 5576 "GET /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg HTTP/1.1" 404 6501
django.request WARNING 2023-08-17 16:02:01,448 log 22052 31484 The request's session was deleted before the request completed. The user may have logged out in a concurrent request, for example.: /uploads/images/orgbatch/GN_0500/GN_0518.jpeg
Traceback (most recent call last):
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.SerializationFailure: could not serialize access due to concurrent update


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\contrib\sessions\backends\db.py", line 86, in save
    obj.save(
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\models\base.py", line 814, in save
    self.save_base(
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\models\base.py", line 877, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\models\base.py", line 990, in _save_table
    updated = self._do_update(
              ^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\models\base.py", line 1054, in _do_update
    return filtered._update(values) > 0
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\models\query.py", line 1231, in _update
    return query.get_compiler(self.db).execute_sql(CURSOR)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\models\sql\compiler.py", line 1984, in execute_sql
    cursor = super().execute_sql(result_type)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\models\sql\compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\backends\utils.py", line 102, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\backends\utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\backends\utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\backends\utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.OperationalError: could not serialize access due to concurrent update


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\contrib\sessions\middleware.py", line 59, in process_response
    request.session.save()
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\contrib\sessions\backends\db.py", line 95, in save
    raise UpdateError
django.contrib.sessions.backends.base.UpdateError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\utils\deprecation.py", line 136, in __call__
    response = self.process_response(request, response)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Local\Miniconda\envs\dj42py311\Lib\site-packages\django\contrib\sessions\middleware.py", line 61, in process_response
    raise SessionInterrupted(
django.contrib.sessions.exceptions.SessionInterrupted: The request's session was deleted before the request completed. The user may have logged out in a concurrent request, for example.
django.server WARNING 2023-08-17 16:02:01,451 basehttp 22052 31484 "GET /uploads/images/orgbatch/GN_0500/GN_0518.jpeg HTTP/1.1" 400 169131
django.request WARNING 2023-08-17 16:03:01,953 log 22052 5576 Not Found: /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg
django.server WARNING 2023-08-17 16:03:01,953 basehttp 22052 5576 "GET /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg HTTP/1.1" 404 6501
django.utils.autoreload INFO 2023-08-17 16:04:54,010 autoreload 11288 28948 Watching for file changes with StatReloader
django.request WARNING 2023-08-17 16:04:58,065 log 11288 28524 Not Found: /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg
django.server WARNING 2023-08-17 16:04:58,068 basehttp 11288 28524 "GET /uploads/images/orgbatch/GN_0500/GN_0517_01.jpeg HTTP/1.1" 404 6501
